packages/fxa-payments-server/src/components/CouponForm/index.tsx

```diff
@@ -226,22 +228,25 @@ export const CouponForm = ({
             </Localized>
           </div>
 
-          <button
-            name="apply"
-            className="button"
-            type="submit"
-            data-testid="coupon-button"
-            disabled={checkingCoupon || readOnly || subscriptionInProgress}
-          >
-            <Localized id="coupon-submit">
-              <span>Apply</span>
-            </Localized>
-          </button>
+          <div className="ml-4">
+            <button
+              name="apply"
+              className="button"
+              type="submit"
+              data-testid="coupon-button"
+              disabled={checkingCoupon || readOnly || subscriptionInProgress}
+            >
+              <Localized id="coupon-submit">Apply</Localized>
+            </button>
+          </div>
         </form>
       )}
       {error && (
         <Localized id={error}>
-          <div className="coupon-error" data-testid="coupon-error">
+          <div
+            className="text-red-700 mt-4 w-[calc(100%-8rem)]"
```

Comment:
`w-[calc(100%-8rem)]`?
With `w-[calc(100%-8rem)]`:
<img width="321" alt="Screen Shot 2022-10-20 at 4 13 19 PM" src="https://user-images.githubusercontent.com/28129806/197048698-c7457038-3213-4057-ad9f-0ba380ac83ad.png">

Without `w-[calc(100%-8rem)]`:
<img width="327" alt="Screen Shot 2022-10-20 at 4 14 07 PM" src="https://user-images.githubusercontent.com/28129806/197048862-5b8e3763-b3a5-4b47-9066-4764e8ef3afd.png">

Reply:
The previous styles on the error included a restricted width to approximately the length of the textfield, but agreed that it looks better taking up the full width. Removed!